src/components/Contacts/ContactsContext/ContactsContext.tsx

```diff
@@ -150,16 +150,16 @@ export const ContactsProvider: React.FC<ContactsContextProps> = ({
     [activeFilters],
   );
 
-  const [contactsView, updateOptions, { loading: userOptionsLoading }] =
+  const [contactsView, saveContactsView, { loading: userOptionsLoading }] =
     useUserPreference({
       key: 'contacts_view',
       defaultValue: TableViewModeEnum.List,
     });
   useEffect(() => {
-    if (contactsView && !userOptionsLoading) {
+    if (contactsView) {
```

Comment:
@dr-bizz When I asked you to add this additional check, I was mistaken. I thought this effect saved the user preference, but it is just updating the view mode in the contacts wrapper. It was right how you had it first, so I'm switching it back.